src/Xrefcheck/Verify.hs

```diff
@@ -465,3 +564,8 @@ checkExternalResource VerifyConfig{..} link
           pure ()
       where
         handler = if secure then withFTPS else withFTP
+
+retryAfterInfo :: Response a -> Maybe (Time Second)
+retryAfterInfo response = case L.lookup "Retry-After" (responseHeaders response) of
```

Comment:
You can use the [built-in constant `hRetryAfter`](https://hackage.haskell.org/package/http-types-0.12.3/docs/Network-HTTP-Types-Header.html#v:hRetryAfter), to protect ourselves against typos.

```suggestion
retryAfterInfo response = case L.lookup hRetryAfter (responseHeaders response) of
```